library/src/main/java/org/apache/apex/malhar/lib/state/managed/ManagedStateImpl.java

```diff
@@ -88,7 +88,7 @@ public void endWindow()
 
   @Min(1)
   @Override
-  public int getNumBuckets()
+  public long getNumBuckets()
```

Comment:
So it is no longer "number of buckets" but "maximum bucket" or what does it represent? There is a setter for number of buckets.